src/components/CTABlock.tsx

```diff
@@ -0,0 +1,49 @@
+import { useState, useCallback } from 'react'
+import { EventControlledContent } from './EventControlledContent'
+import { mathifyElement } from '../lib/math'
+
+interface CTABlockProps {
+  contentString: string
+  contentPrompt: string
```

Comment:
Minor: `contentString` seems a bit redundant (e.g. including the type in the variable name). Maybe this can just be `content` and then `contentPrompt` --> `prompt` for consistency relative to that?